src/Paysera/WalletApi/Mapper.php

```diff
@@ -2387,4 +2401,56 @@ public function encodeTransferConfiguration(Paysera_WalletApi_Entity_TransferCon
     {
         return (new Paysera_WalletApi_Mapper_TransferConfigurationMapper())->mapFromEntity($transferConfiguration);
     }
+
+    /**
+     * @param array $data
+     * @return Paysera_WalletApi_Entity_ClientPermissionsToWallet[]
+     */
+    public function decodeClientPermissionsToWallets(array $data)
```

Comment:
it would be better, if besides of using generic variables like array/list/string/data you would be using variable, so in the specific case it should be $clientPermissionsToWallet

Reply:
Done